scrapi/harvesters/addis_ababa.py

```diff
@@ -6,6 +6,27 @@
 from __future__ import unicode_literals
 
 from scrapi.base import OAIHarvester
+from scrapi.base import helpers
+
+
+def format_uris_addis_ababa(*args):
+    identifiers = helpers.gather_identifiers(args)
+    provider_uris, object_uris = helpers.seperate_provider_object_uris(identifiers)
+
+    for arg in args:
+        if arg and 'http://hdl.handle.net/123456789/' in arg[0]:
+            doc_id = arg[0].replace('http://hdl.handle.net/123456789/', '')
+            canonical_uri = 'http://etd.aau.edu.et/handle/123456789/' + doc_id
+            provider_uris = ['http://etd.aau.edu.et/handle/123456789/' + doc_id]
```

Comment:
Here is where you may be turning `provider_uris` from a list with many elements to a list with one element. Also, it doesn't look like you are using the `provider_uris` that you make above, does it not pull out the right information?